Calculator/Calculator/ViewController.swift

```diff
@@ -8,56 +8,153 @@
 
 import UIKit
 
-class ViewController: UIViewController {
-
-    @IBOutlet weak var resultTextField: UITextField!
-    var result = Float();
-    var currentNumber = Float();
-    var currentOperator: String!;
+class ViewController: UIViewController, UITextFieldDelegate  {
     
+    @IBOutlet weak var expressionTextField: UITextField!
+    @IBOutlet weak var resultTextField: UITextField!
+    let kLimit: Float = -100000000.0
```

Comment:
No need to change in this case. 
Howerver, in case data type is implicit, specification is necessary.

// ◯ Int / UInt / NSInteger / NSUInteger these data types should be distinguished so specification is necessary
let i: Int = 0

// ◯ Float / CGFloat / Double these data types should be distinguished so specification is necessary
let p: Float = 3.14